Test/Unit/Model/RunnerTest.php

```diff
@@ -58,33 +58,54 @@ public function testRun()
             'diff -r Magento-EE-2.3.1/vendor/magento/module-checkout/view/frontend/web/js/view/billing-address.js Magento-EE-2.3.3/vendor/magento/module-checkout/view/frontend/web/js/view/billing-address.js',
             'diff -r Magento-EE-2.3.1/vendor/magento/module-bundle/view/frontend/templates/js/components.phtml Magento-EE-2.3.3/vendor/magento/module-bundle/view/frontend/templates/js/components.phtml',
             'diff -r Magento-EE-2.3.1/vendor/magento/module-checkout/view/frontend/web/template/billing-address/details.html Magento-EE-2.3.3/vendor/magento/module-checkout/view/frontend/web/template/billing-address/details.html',
+            'diff -r Magento-EE-2.3.1/vendor/magento/module-customer/view/frontend/templates/form/forgotpasswords.phtml Magento-EE-2.3.3/vendor/magento/module-customer/view/frontend/templates/form/forgotpasswords.phtml',
```

Comment:
Build is failing for me locally. I think this all instances should be `forgotpassword.phtml` here (not `forgorpasswords.phtml`)

Reply:
Updated this as well